independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/buildtool/maven/base/pom.tpl.qute.xml

```diff
@@ -118,6 +118,9 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>$\{compiler-plugin.version}</version>
+                <configuration>
+                    <parameters>$\{maven.compiler.parameters}</parameters>
```

Comment:
Maybe we should render a comment what this is for? Otherwise it may look strange/inconsistent when you don't know the details.

Reply:
I am not sure we should add comment for this, else we would need to comment the whole pom.xml :)

Reply:
Perhaps a minor comment with the https://youtrack.jetbrains.com/issue/IDEA-201496 link would be nice to have, just so we can remember to remove it when that bug is fixed :)

Reply:
@gastaldi I added it, but it seems they are deleted during the smart pom merge :-/

Reply:
Hm, I think the maven-model-merger ignores comments from the non-dominant side. 

I'd say that maybe having the comment in the template should be enough for now until we fix this. 

Looking at the other side, also I don't see a reason that we'll want this parameter to be false, so maybe removing the `maven.compiler.parameters` property and make it fixed to `true` directly in the plugin configuration should be good enough ™️ :)